doc/devdocs/development/new-powertoy.md

```diff
@@ -0,0 +1,294 @@
+# 🧭 Creating a New PowerToy: End-to-End Developer Guide
+
+First of all, thank you for wanting to contribute to PowerToys. The work we do would not be possible with out the support of community supporters like you.
+
+This guide documents the process of building a new PowerToys utility from scratch, including architecture decisions, integration steps, and common pitfalls.
+
+---
+
+## 1. Overview and Prerequisites
+
+A PowerToy module is a self-contained utility integrated into the PowerToys ecosystem. It can be UI-based, service-based, or both.
+
+### Requirements
+
+* [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) and the following workloads/individual components
```

Comment:
Total nit: should we call out that Community is fine? "Community or better" sounds weird, but you get the point